src/components/molecules/about-header/about-header.tsx

```diff
@@ -11,13 +11,13 @@ export default function AboutHeader() {
     >
         <div className="w-full flex flex-col items-center justify-center px-[18px] lg:px-0 bg-gradient-to-r from-black/60  to-transparent gap-4">
             
-            <HeaderText level={"h1"} className={"text-white"} fontWeight={"extrabold"} fontSize={{base: "39px", md:"56px"}}>
+            <HeaderText data-cy={"about-header-header"} level={"h1"} className={"text-white"} fontWeight={"extrabold"} fontSize={{base: "39px", md:"56px"}}>
                 Transnational <span className="text-[#7AB8F1]">Job Listing</span> Channel
             </HeaderText>
 
             <div className="text-white flex justify-start items-start lg:px-5 w-full lg:w-[40%] text-left lg:text-center">
               <p className="">
-                So many jobs available, all you have to do is keep up with  our posts. Check below for recent job openings. 
+                So many jobs available, all you have to do is keep up with our posts. Check below for recent job openings. 
```

Comment:
Removed extraneous whitespace as it was messing up the cy.contains check (oddly should('contains', ...) was fine with it)